### PR TITLE
One missing Bitmap to rule them all

### PR DESCRIPTION
**Pull request by AbandonedCart**
`... and one condition to find them without the need for multiple instances and passing a condition. The data class can simply return the same instance every time it's necessary.`

Jokes aside, there are a minimum of 3 instances of the missingIcon Bitmap per run and it's a static icon. If the explanation isn't convincing enough, compare the load times.

### CODE REVIEW

**Review by lynxnb — APPROVED** [review]
LGTM, but maybe reword the commit title to something more useful long term. We like funny stuff in our commit descriptions, not titles :)

**Comment by AbandonedCart** [comment]
Boring version, done.
